app/javascript/miq-redux/store.ts

```diff
@@ -0,0 +1,19 @@
+import { createStore, applyMiddleware, compose, Store} from 'redux';
+import {rootReducer} from './reducer';
+
+import {AppState} from '../packs/miq-redux';
+
+const composeEnhancers = window['.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__'] || compose;
```

Comment:
Because typescript is sometimes too aggressive and was complaining that `__REDUX_DEVTOOLS_EXTENSION_COMPOSE__` does not exists on window. 

But since you brought it up, which one would you prefer? Because I am Indecisive between accessing it via [] or re-typing window object to any.
```typescript
const composeEnhancers = (<any>window).__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```